docs/programming-guide.md

```diff
@@ -1144,9 +1144,11 @@ generate these on the reduce side. When data does not fit in memory Spark will s
 to disk, incurring the additional overhead of disk I/O and increased garbage collection.
 
 Shuffle also generates a large number of intermediate files on disk. As of Spark 1.3, these files
```

Comment:
I agree it could be removed too, even if it probably doesn't matter at this point since we are well beyond 1.3.